admin_manual/configuration_server/occ_command.rst

```diff
@@ -199,8 +199,22 @@ keys after upgrading your ownCloud server::
 File Operations
 ---------------
 
-The ``files:scan`` command scans for new files for the file cache, and isn't 
-intended to be run manually.
+The ``files:scan`` command scans for new files and updates the file cache. You 
+may rescan all files, per-user, a space-delimited list of users, and limit the 
+search path::
+
+ $ sudo -u www-data php owncloud/occ  files:scan --help
```

Comment:
Can we please be consistent and use `occ` instead of `owncloud/occ`, because the command listings above also use this.